api/resolvers/user.js

```diff
@@ -567,6 +562,31 @@ export default {
         return true
       }
 
+      const invoiceActionFailed2 = await models.invoice.findFirst({
```

Comment:
I struggled with writing a good comment since I already struggled with grasping this query, lol.

I think this didn't break anything but I didn't fully QA everything again.